components/datadog/agent/kubernetes.go

```diff
@@ -0,0 +1,55 @@
+package agent
+
+import (
+	"github.com/pulumi/pulumi-kubernetes/sdk/v3/go/kubernetes"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+
+	"github.com/DataDog/test-infra-definitions/common/config"
+	"github.com/DataDog/test-infra-definitions/components"
+	"github.com/DataDog/test-infra-definitions/components/datadog/kubernetesagentparams"
+)
+
+type KubernetesAgentOutput struct {
+	components.JSONImporter
+
+	AgentInstallNameLinux   string `json:"agentInstallNameLinux"`
```

Comment:
The fields are already part of the `KubernetesAgent` struct, adding `agent` is redundant.